16_wordle/main.go

```
@@ -48,6 +39,21 @@ func main() {
 	}
 }
 
+func NewList(path string) (List, error) {
+	rawWordList, err := os.ReadFile(path)
+
+	if err != nil {
+		log.Fatal(err)
+	}
```

you might want to return the error here

---

I'll read about error handling. But added what I assume to makes sense, since I do have return a `List` and can't just return the error I think.